Zombie Defense DTM AD/map.xml

```diff
@@ -1,275 +1,326 @@
+    <complement id="fort-roof">
+      <cuboid min="-29,27,-29" max="29,28,29"/>
+      <cuboid min="-20,27,-20" max="20,28,20"/>
+    </complement>
+    <union id="zombies-notube">
+      <negative>
+        <complement>
+          <cuboid min="-87,50,-87" max="87,57,87"/>
+          <cuboid min="-67,50,-67" max="67,57,67"/>
+        </complement>
+      </negative>
+    </union>
+    <union id="steves-nohead">
+      <negative>
+        <cuboid min="-8,53,-8" max="8,67,8"/>  
+      </negative>
+    </union>
+    <union id="noexplosion">
+      <cuboid min="-20,-oo,-20" max="20,21,20"/>
+      <region id="noteamgrief-z"/>
+      <region id="wooden deck"/>
+      <region id="fort-roof"/>
+      <cuboid min="-30,20,-30" max="30,21,30"/>
+      <cuboid min="-30,5,-30" max="30,6,30"/>
+    </union>
+    <cuboid id="release" min="-86,50,-86" max="86,51,86"/>
+    
+    <apply block="never" message="Oops! This is uneditable area!" region="uneditable-area"/>
+    <apply enter="only-zombies" message="Hey! This is a `2Zombies`c-only area!" region="zombies-only"/>
+    <apply enter="only-steves" message="Hey! This is a `6Steves`c-only area!" region="steves-only"/>
+    <apply kit="zombie_notube" filter="only-zombies" region="zombies-notube"/>
+    <apply kit="steve_noinhead" filter="only-steves" region="steves-nohead"/>   
+    <apply block-break="never" message="Oops! This block is unbreakable!" region="unbreakable"/>
+    <apply block="explosionprotect" region="noexplosion"/>    
+    <apply block="noteamgrief-steves" region="noteamgrief-s"/>
+    <apply block="noblowupchest-steves" region="grass area rec"/>
+  </regions>
+  <itemremove>
+    <item>iron_sword</item>
+    <item>iron_boots</item>
+    <item>iron_axe</item>
+    <item>diamond_pickaxe</item>
+    <item>bow</item>
+    <item>skull_item</item>
+    <item>arrow</item>
+    <item>log</item>
+    <item>log 2</item>
+    <item>vine</item>
+    <item>leather_chestplate</item>
+    <item>leather_leggings</item>
+    <item>redstone</item>
+    <item>redstone_block</item>
+    <item>redstone_torch_on</item>
+    <item>smooth_brick</item>
+    <item>nether_fence</item>
+    <item>stone_plate</item>
+    <item>stone_button</item>
+    <item>stained_glass</item>
+    <item>step</item>
+    <item>water_bucket</item>
+    <item>bucket</item>
+    <item>wood</item>
+    <item>wood_button</item>
+    <item>tnt</item>
+  </itemremove>
+  <portals>
+    <!--Ground-->
+    <portal region="ground-f" destination="ground-d" yaw="0"/>
+    <!--Basement-->
+    <portal region="basement-f" destination="basement-d" yaw="0"/>
+  </portals>
+  <broadcasts>
+    <alert after="0s">`6Steves! You have 30 seconds preparation time! Prepare for `2Zombies `cattack`6!</alert>
+    <tip after="1s" every="2m">Killrewards are 16 Arrows,16 TNTs,and Golden apple.</tip>
+    <tip after="2s" every="2m">You can get enchanted Iron_chestplate on uneditable column islands!</tip>
+    <alert after="30s">`2Zombies `creleased!</alert>
```

Comment:
``` `6Steves have 30 seconds to prepare. Prepare for the `2Zombie `cattack`6! ```
`Killrewards are 16 arrows, 16 TNT and a golden apple.`
`You can get enchanted iron chestplates on the column islands!` Maybe column -> corner?